db/models/rooms.js

```diff
@@ -3,7 +3,8 @@ const { Schema } = mongoose;
 const db = require('../../db');
 
 const roomsSchema = new Schema({
-  // reservations_id: { type: Schema.Types.ObjectId, ref: 'Reservations', required: true },
+  reservations_id: { type: String },
```

Comment:
This one is tricky because it doesn't always have a reservation id. So in reservations side I had to make it type mixed.  We will just have to be careful if we add/modify to convert to ObjectId when there is a reservation_id and then insert an empty string when there is no reservation_id

Reply:
okay i used to put ObjectId before and i changed it to string because reservation_id is sometime empty string when there is no reservation. 
mixed sounds like a good solution.